src/org/purefn/river/batch.clj

```diff
@@ -0,0 +1,137 @@
+(ns org.purefn.river.batch
+  "A consumer suitable for batch reading operations from Kafka."
+  (:require [clojure.pprint :as pprint]
+            [clojure.spec.alpha :as s]
+            [com.stuartsierra.component :as component]
+            ;; temp, remove
```

Comment:
Why "temp"? Remove when?

Reply:
I don't want a hard dependency on `nippy`.  Soon as I get to it :)